### ISSUE TITLE
Create users and assign event type programmatically

### ISSUE THREAD

**Issue by harshajayaweeraXHJ** [calcom/cal.com#517]
Hi! All. I have being working with Calendso for few weeks now and it's being great. I have some question about it's API.
I have an app where users can book experts for consultations. For that I need to create multiple Users (for experts) in backend and embed their calendar in each experts page so each expert has separate calendar urls
I can do it already with `/api/auth/signup` endpoint. 

But it to be usable and show via calendar I need to add event type for each users. How can I do this via an API. I see that Calendso has `/api/availability/eventtype`  endpoint but it I think require authentication. How do I get authentication credentials for accessing API programmatically.
Basically what I want to do it Create users programmatically and assign event type for them

**Comment by sean-brydon** [comment]
This ticket is being close due to inactivity - This sounds like it could be a use case for our API https://developer.cal.com/
